src/OrchardCore/OrchardCore.DisplayManagement/Handlers/DisplayDriver.cs

```diff
@@ -13,6 +13,7 @@ public abstract class DisplayDriver<TModel, TDisplayContext, TEditorContext, TUp
         where TEditorContext : BuildEditorContext
         where TUpdateContext : UpdateEditorContext
     {
+        protected string PrefixRoot { get; set; } = typeof(TModel).Name;
```

Comment:
No need to change this if you modify LayerMetadataWelder as above

Reply:
@agriffard  Please try it and change like this if that works. Looks safer (in terms of not not breaking anything else)